homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -218,6 +233,17 @@ async def async_step_device_config(
             _LOGGER.exception(exception)
             errors["base"] = "unknown"
         else:
+            if self._reauth_entry:
+                self.hass.config_entries.async_update_entry(
+                    self._reauth_entry,
+                    data={
+                        CONF_WEBFSAPI_URL: self._webfsapi_url,
+                        CONF_PIN: user_input[CONF_PIN],
+                    },
```

Comment:
I don't think you should update the URL:
```suggestion
                    data={CONF_PIN: user_input[CONF_PIN]},
```